docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -13,32 +13,19 @@ weight: 200
 
 # Configure generic OAuth authentication
 
-You can configure many different OAuth2 authentication services with Grafana using the generic OAuth2 feature. Examples:
-
-- [Configure generic OAuth authentication](#configure-generic-oauth-authentication)
-  - [Email address](#email-address)
-  - [Groups / Teams](#groups--teams)
-  - [Login](#login)
-  - [PKCE](#pkce)
-  - [Configure refresh token](#configure-refresh-token)
-  - [Configure automatic login](#configure-automatic-login)
-  - [Set up OAuth2 with Auth0](#set-up-oauth2-with-auth0)
-  - [Set up OAuth2 with Bitbucket](#set-up-oauth2-with-bitbucket)
-  - [Set up OAuth2 with Centrify](#set-up-oauth2-with-centrify)
-  - [Set up OAuth2 with OneLogin](#set-up-oauth2-with-onelogin)
-  - [Role Mapping](#role-mapping)
-    - [JMESPath examples](#jmespath-examples)
-      - [Map user organization role](#map-user-organization-role)
-      - [Map server administrator privileges](#map-server-administrator-privileges)
-  - [Team synchronization](#team-synchronization)
-  - [Skip organization role sync](#skip-organization-role-sync)
+Grafana has specialised OAuth2 integrations for the following auth providers:
 
-This callback URL must match the full HTTP address that you use in your browser to access Grafana, but with the suffixed path of `/login/generic_oauth`.
+- [Azure AD OAuth]({{< relref "../azuread" >}})
+- [GitHub OAuth]({{< relref "../github" >}})
+- [GitLab OAuth]({{< relref "../gitlab" >}})
+- [Google OAuth]({{< relref "../google" >}})
+- [Grafana Com OAuth]({{< relref "../grafana-com" >}})
+- [KeyCloak OAuth]({{< relref "../keycloak" >}})
+- [Okta OAuth]({{< relref "../okta" >}})
 
-You may have to set the `root_url` option of `[server]` for the callback URL to be
-correct. For example in case you are serving Grafana behind a proxy.
+If your OAuth2 provider is not among them, you can use generic OAuth authentication to integrate it with Grafana.
 
-Example config:
+## Example configuration
```

Comment:
Why is this example here? Without any kind of introduction content to explain what I'm seeing, this seems kind of random.